server/routes/bookings/bookingsController.test.ts

```diff
@@ -40,8 +42,6 @@ describe('Bookings homepage (future visits list)', () => {
         expect($('title').text()).toMatch(/^Bookings -/)
         expect($('[data-test="back-link"]').length).toBe(0)
         expect($('h1').text()).toBe('Bookings')
-        expect($('h1').siblings('h2').first().text()).toBe('Future visits')
```

Comment:
should this assertion be removed?

        expect($('h1').siblings('h2').first().text()).toBe('Future visits')